Formula/h/hyperestraier.rb

```diff
@@ -4,6 +4,7 @@ class Hyperestraier < Formula
   url "https://dbmx.net/hyperestraier/hyperestraier-1.4.13.tar.gz"
   sha256 "496f21190fa0e0d8c29da4fd22cf5a2ce0c4a1d0bd34ef70f9ec66ff5fbf63e2"
   license "LGPL-2.1-or-later"
+  revision 1
```

Comment:
Will drop this after verifying Linux CI

Reply:
Seems to work https://github.com/Homebrew/homebrew-core/actions/runs/14575235740/job/40880121887?pr=220560#step:4:87